src/js/components/TaskDetail.js

```diff
@@ -287,37 +353,66 @@ class TaskSidePanelContents extends SidePanelContents {
     super.tabs_handleTabClick(...arguments);
   }
 
+  getNotFound(itemType) {
```

Comment:
Why the premature generalisation here? It seems to me we only ever call this with `task` as the `itemType`. If we have concrete use cases then I'd also like to see the `taskID` being an argument so that we can more easily extract this whole thing into an own component if appropriate.